# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium-shim/patches/with_update/patch_order.txt
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 3
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium-shim/patches/with_update/patch_order.txt
--- a/security/sandbox/chromium-shim/patches/with_update/patch_order.txt	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium-shim/patches/with_update/patch_order.txt	Wed Jun 12 11:10:48 2019 +0100
@@ -1,4 +1,3 @@
-apply_upstream_alternate_desktop_fix.patch
 update_chromium_linux_x86_syscalls.patch
 revert_remove_AddTargetPeer.patch
 revert_remove_BrokerDuplicateHandle.patch
@@ -6,19 +5,21 @@
 ifdef_out_FromStringInternal.patch
 add_option_to_not_use_restricting_sids.patch
 ifdef_out_SequenceChecker_code.patch
-revert_c++14_typename_alias_usage.patch
-revert_c++14_index_sequence_usage.patch
-fix_Wcomma_warning_in_time_cc.patch
 allow_read_only_all_paths_rule.patch
 revert_TargetNtSetInformationThread_change.patch
 mingw_base_win_get_caller.patch
-mingw_duplicate_instatinations.patch
 mingw_copy_s.patch
 mingw_operator_new.patch
 mingw_cast_getprocaddress.patch
 mingw_capitalization.patch
 mingw_disable_one_try.patch
-mingw_noexports_casts.patch
 mingw_offsetof.patch
-add_aarch64_windows_support.patch
+fix_incorrect_int_use_in_Kernel32BaseVersion.patch
+revert_removal_of_AlterEnvironment_on_Windows.patch
+allow_env_changes.patch
+ifdef_out_AppContainerProfileBase_testing_functions.patch
+mingw_missing_windows_types_defines.patch
+mingw_undefine_MemoryBarrier.patch
+add_return_in_QueryCancellationTraitsForNonCancellables_to_satisfy_build.patch
+include_atomic_header_in_platform_thread.patch
 aarch64_control_flow_guard.patch